releases.yml

```diff
@@ -3,6 +3,13 @@
     - component: Common grid patterns
       url: /docs/patterns/grid#common-grid-patterns
       status: New
+    - component: Rule
+      url: /docs/patterns/rule
+      status: New
+      notes: We've added a new horizontal bar component.
+    - component: Vertical spacing
```

Comment:
This again is creeping back. Vertical spacing has been removed from `main` branch already.

Reply:
Damnit!